zhaquirks/thirdreality/vibrate.py

```diff
@@ -0,0 +1,66 @@
+"""Third Reality vibrate devices."""
+from zigpy.quirks import CustomDevice
+from zigpy.zcl.clusters.general import Basic, Ota, PowerConfiguration
+from zhaquirks import CustomCluster
+from zigpy.zcl.clusters.security import IasZone
+import zigpy.types as t
+
+from zhaquirks.const import (
+    DEVICE_TYPE,
+    ENDPOINTS,
+    INPUT_CLUSTERS,
+    MODELS_INFO,
+    OUTPUT_CLUSTERS,
+    PROFILE_ID,
+)
+from zhaquirks.thirdreality import THIRD_REALITY
+
+MANUFACTURER_SPECIFIC_CLUSTER_ID = 0xFFF1
+
+class ThirdRealityAccelCluster(CustomCluster):
+    """ThirdReality Acceleration Cluster."""
+
+    cluster_id = MANUFACTURER_SPECIFIC_CLUSTER_ID
+    attributes = {
+        0x0001: ("x_axis", t.int16s, True),
+        0x0002: ("y_axis", t.int16s, True),
+        0x0003: ("z_axis", t.int16s, True),
+    }
```

Comment:
1. So for now, these attributes could only be read using the "clusters UI" in ZHA, right?
2. Do you plan on exposing these custom attributes in the HA UI [by adding a new match/sensor in ZHA (at the bottom of the file)](https://github.com/home-assistant/core/blob/dev/homeassistant/components/zha/sensor.py)? (And also binding the cluster by adding it to the ["manufacturer specific" cluster handler](https://github.com/home-assistant/core/blob/dev/homeassistant/components/zha/core/cluster_handlers/manufacturerspecific.py)?
3. Does the `IasZone` cluster on this device do anything? So "on/off" for vibrations?

Reply:
now, these attributes could only be read using the "clusters UI" in ZHA is ok, we will try exposing these custom attributes in the HA UI later.
About Iaszone cluster, when the device vibrate, will send accelcluster and iaszone to show the vibration